openweathermap-ios/openweathermap-ios/Weather/Model/WeatherData.swift

```diff
@@ -0,0 +1,14 @@
+
+struct WeatherData: Codable {
+    let name: String
+    let main: Main
+    let weather: [Weather]
+}
+
+struct Main: Codable {
+    let temp: Double
+}
+
+struct Weather: Codable {
```

Comment:
Consider using embedded structures for those codable used to parse responses.

Reply:
The objective of the "WeatherModel" is to have a structure with the most varied climatic conditions. This link shows the different "ids" for each weather condition.

https://openweathermap.org/weather-conditions

I think it makes more sense now
![image](https://user-images.githubusercontent.com/21261476/79124841-8b4cd200-7d73-11ea-9d41-96b84d4c3baf.png)

If you are still confused what can be done to improve?

Reply:
I didn't understand what to do here.

Reply:
```swift
extension WeatherData {
    struct Main: Codable { ... }
    struct Weather: Codable { ... } 
}
```